### DISCUSSION TITLE
В одно поле, 2 поля из другой таблицы

### DISCUSSION THREAD

**Discussion by Leo-ui-528** [totumonline/totum-ru-issues-and-discussions#260] [Как это сделать?] [answered]
Подскажите пожалуйста, как в одно поле вставить 2 поля из другой таблицы?

**Comment by ttm-alexeypolunin** [comment]
Что-то непонятное спрашиваете — уточните

**Comment by Leo-ui-528** [comment, marked as answer]
Есть таблица,  поручения в которой 2 поля номер и дата а в таблице документ мне нужно в одном поле собрать  2  в которой будет написан и номер и дата. Селектом так не получается.

**Reply by ttm-alexeypolunin** [reply]
В Документ (предположим он тип Строка) берем row из номера и даты (чтоб один запрос был) и склеиваем:

```
=: str`$row[num] ++ $row[date]`

~row: selectRow(table: 'some_table'; field: 'num'; field: 'date')

// тильда у строки row обязательна — она дает команду при втором обращении не выполнять новый запрос
```

А можно и просто двумя select, нагрузка не очень большая.

**Reply by ttm-alexeypolunin** [reply]
Если речь идет про поле Селект, чтобы в отображении Селекта было несколько полей то вот обсуждали: https://github.com/totumonline/totum-ru-issues-and-discussions/discussions/228

**Comment by Leo-ui-528** [comment]
Спасибо
